nilearn/plotting/matrix_plotting.py

```diff
@@ -264,7 +264,6 @@ def plot_contrast_matrix(contrast_def, design_matrix, colorbar=False, ax=None,
         Include a colorbar in the contrast matrix plot. Default=False.
 
     ax : matplotlib Axes object, optional
```

Comment:
Maybe add here a minimalistic description here (for people not familiar with mpl).